src/app/Plans/components/Wizard/helpers.tsx

```diff
@@ -691,6 +705,20 @@ export const useEditingPlanPrefillEffect = (
   };
 };
 
+export const getClonedPlanDefaultName = (
+  planBeingPrefilled: IPlan,
+  plansQuery: UseQueryResult<IKubeList<IPlan>>
+) => {
+  let name = `Copy of ${planBeingPrefilled.metadata.name}`;
+  // Make sure the new name is also unique; if it is already in use put a number at the end of it.
```

Comment:
Not sure the comment is needed if it's a remnant of the development.
Meanwhile if you feel it helps I'd make it clearer and at top of function.
For instance "Name is unique. Add a number if name is already used"

Reply:
I think this is another case of me over-commenting. I'll remove it.